tests/unit_tests/plugin_tests/test_grouptree.py

```diff
@@ -1,17 +1,26 @@
 import datetime
+from pathlib import Path
+from typing import List, Tuple
 
 import pandas as pd
 import pytest
+from _pytest.fixtures import SubRequest
 
-from webviz_subsurface.plugins._group_tree.group_tree_data import add_nodetype_for_ens
+from webviz_subsurface._providers.ensemble_summary_provider._provider_impl_arrow_presampled import (
+    ProviderImplArrowPresampled,
+)
+from webviz_subsurface._providers.ensemble_summary_provider.ensemble_summary_provider import (
+    EnsembleSummaryProvider,
+)
```

Comment:
`from webviz_subsurface._providers import EnsembleSummaryProvider` Should be enough?